index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+   <header>
+      <h1 id="title">Albert Einstein</h1>
+      <p>Albert Einstein was born in 1879 in Germany, the first child of a bourgeois Jewish couple. The young Albert displayed an early interest in science, but he was unhappy with the principles of obedience and conformity that governed his Catholic elementary school.</p>
+    </header>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <div id="title">
+        Albert Einstein
+      </div>
+      <div id="img-div">
+        <img src="" alt="" id="image" style="max-width:100%;display:block;" height="">
+        <div id="img-caption">
+		<div id="img-div">
+      <img src="https://sneakzz.github.io/FreeCodeCamp/Responsive%20Web%20Design%20Projects/Tribute%20page/images/Einstein.jpg">
+      <p id="img-caption">Albert Einstein gazing, thinking about a complicated physics formula.</p>
+    </div>
```

Comment:
I think it would be better to use a code formatter, to make the code look more organized, and professional.